Description: Use the -g1 debugging level instead of -g when compiling WebKit
 to reduce the memory used and avoid build failures caused by lack of RAM.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
Taken from: https://sources.debian.org/patches/openjfx/11.0.2+1-1/25-webkit-debug-level.patch
--- a/modules/javafx.web/src/main/native/Source/cmake/WebKitCompilerFlags.cmake
+++ b/modules/javafx.web/src/main/native/Source/cmake/WebKitCompilerFlags.cmake
@@ -89,6 +89,7 @@
 
 if (COMPILER_IS_GCC_OR_CLANG)
     WEBKIT_APPEND_GLOBAL_COMPILER_FLAGS(-fno-strict-aliasing)
+    WEBKIT_APPEND_GLOBAL_COMPILER_FLAGS(-g1)
 
     # clang-cl.exe impersonates cl.exe so some clang arguments like -fno-rtti are
     # represented using cl.exe's options and should not be passed as flags, so

